### ISSUE TITLE
GluonTS example airpassengers dataset not found

### ISSUE THREAD

**Issue by RashidBakirov** [awslabs/gluon-ts#2461]
am trying to run the GluonTS example code, going through some struggle to install the libraries, now I get the following error:

`FileNotFoundError: C:\Users\abcde\.mxnet\gluon-ts\datasets\airpassengers\test`

The `C:\Users\abcde\.mxnet\gluon-ts\datasets\airpassengers\` does exist but contains only `train` folder. Have tried reinstalling but to no avail. Any ideas how to fix this and run the example, even if finding the dataset in correct format elsewhere?

**Comment by lostella** [comment]
@RashidBakirov what version of gluonts are you running?

**Comment by RashidBakirov** [comment]
`gluonts 0.11.3` installed with
`pip install "gluonts[mxnet,pro]"`
`pip install "gluonts[torch,pro]"`

**Comment by jaheba** [comment]
This appears to be an issue with `PandasDataset`.

```py
>>> test = PandasDataset(df, target="#Passengers")
>>> list(test)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/schjaspe/.virtualenvs/fresh/lib/python3.9/site-packages/gluonts/dataset/pandas.py", line 173, in __iter__
    for entry in self._data_entries:
  File "/Users/schjaspe/.virtualenvs/fresh/lib/python3.9/site-packages/gluonts/dataset/pandas.py", line 148, in _pair_to_dataentry
    df = df.to_period(freq=self.freq)
  File "/Users/schjaspe/.virtualenvs/fresh/lib/python3.9/site-packages/pandas/core/frame.py", line 11494, in to_period
    new_ax = old_ax.to_period(freq=freq)
  File "/Users/schjaspe/.virtualenvs/fresh/lib/python3.9/site-packages/pandas/core/indexes/extension.py", line 98, in method
    result = attr(self._data, *args, **kwargs)
  File "/Users/schjaspe/.virtualenvs/fresh/lib/python3.9/site-packages/pandas/core/arrays/datetimes.py", line 1137, in to_period
    return PeriodArray._from_datetime64(self._ndarray, freq, tz=self.tz)
  File "/Users/schjaspe/.virtualenvs/fresh/lib/python3.9/site-packages/pandas/core/arrays/period.py", line 298, in _from_datetime64
    data, freq = dt64arr_to_periodarr(data, freq, tz)
  File "/Users/schjaspe/.virtualenvs/fresh/lib/python3.9/site-packages/pandas/core/arrays/period.py", line 1066, in dt64arr_to_periodarr
    base = freq._period_dtype_code
AttributeError: 'pandas._libs.tslibs.offsets.MonthBegin' object has no attribute '_period_dtype_code'
```

**Comment by jaheba** [comment]
Rather, it's an issue with pandas -- how surprising!

I've submitted a fix in #2462

**Comment by jaheba** [comment]
This will be fixed in `0.11.4`.

You can install the fixed version from GitHub until then:

```
pip install -U git+https://github.com/awslabs/gluonts.git
```

**Comment by lostella** [comment]
> Rather, it's an issue with pandas -- how surprising!
> 
> I've submitted a fix in #2462

Should an issue be opened to pandas?

**Comment by RashidBakirov** [comment]
> This appears to be an issue with `PandasDataset`.
> 
> ```python
> >>> test = PandasDataset(df, target="#Passengers")
> >>> list(test)
> Traceback (most recent call last):
>   File "<stdin>", line 1, in <module>
>   File "/Users/schjaspe/.virtualenvs/fresh/lib/python3.9/site-packages/gluonts/dataset/pandas.py", line 173, in __iter__
>     for entry in self._data_entries:
>   File "/Users/schjaspe/.virtualenvs/fresh/lib/python3.9/site-packages/gluonts/dataset/pandas.py", line 148, in _pair_to_dataentry
>     df = df.to_period(freq=self.freq)
>   File "/Users/schjaspe/.virtualenvs/fresh/lib/python3.9/site-packages/pandas/core/frame.py", line 11494, in to_period
>     new_ax = old_ax.to_period(freq=freq)
>   File "/Users/schjaspe/.virtualenvs/fresh/lib/python3.9/site-packages/pandas/core/indexes/extension.py", line 98, in method
>     result = attr(self._data, *args, **kwargs)
>   File "/Users/schjaspe/.virtualenvs/fresh/lib/python3.9/site-packages/pandas/core/arrays/datetimes.py", line 1137, in to_period
>     return PeriodArray._from_datetime64(self._ndarray, freq, tz=self.tz)
>   File "/Users/schjaspe/.virtualenvs/fresh/lib/python3.9/site-packages/pandas/core/arrays/period.py", line 298, in _from_datetime64
>     data, freq = dt64arr_to_periodarr(data, freq, tz)
>   File "/Users/schjaspe/.virtualenvs/fresh/lib/python3.9/site-packages/pandas/core/arrays/period.py", line 1066, in dt64arr_to_periodarr
>     base = freq._period_dtype_code
> AttributeError: 'pandas._libs.tslibs.offsets.MonthBegin' object has no attribute '_period_dtype_code'
> ```

I was getting this error as well, but somehow reinstalling gluonts several times seem to have fixed it. 

> This will be fixed in `0.11.4`.
> 
> You can install the fixed version from GitHub until then:
> 
> ```
> pip install -U git+https://github.com/awslabs/gluonts.git
> ```

Thank you. I will give it a go.

**Comment by lostella** [comment]
> I was getting this error as well, but somehow reinstalling gluonts several times seem to have fixed it.

computers!

**Comment by RashidBakirov** [comment]
> This will be fixed in `0.11.4`.
> 
> You can install the fixed version from GitHub until then:
> 
> ```
> pip install -U git+https://github.com/awslabs/gluonts.git
> ```

Now I am getting the following error:

```
ModuleNotFoundError                       Traceback (most recent call last)
Cell In [1], line 5
      3 from gluonts.dataset.pandas import PandasDataset
      4 from gluonts.dataset.repository.datasets import get_dataset
----> 5 from gluonts.model.deepar import DeepAREstimator
      6 from gluonts.mx import Trainer
      8 dataset = get_dataset("airpassengers")

ModuleNotFoundError: No module named 'gluonts.model.deepar'
```

Also, no test folder in airpassengers again :(

**Comment by jaheba** [comment]
Looks like the website is outdated (here is the fix: https://github.com/awslabs/gluonts/pull/2467).

You now do

```py
from gluonts.mx import DeeparEstimator
```

Have you tried using `regenerate=True`?

**Comment by RashidBakirov** [comment]
> Looks like the website is outdated (here is the fix: #2467).
> 
> You now do
> 
> ```python
> from gluonts.mx import DeeparEstimator
> ```
Thanks for your continuous assistance, but this is still not working :(

```ImportError: cannot import name 'DeeparEstimator' from 'gluonts.mx' (C:\Users\abcde\Anaconda3\envs\gluontsenv\lib\site-packages\gluonts\mx\__init__.py)```

**Comment by melopeo** [comment]
Maybe try the following:

`from gluonts.mx.model.deepar import DeepAREstimator`

**Comment by jaheba** [comment]
> > Looks like the website is outdated (here is the fix: #2467).
> 
> > 
> 
> > You now do
> 
> > 
> 
> > ```python
> 
> > from gluonts.mx import DeeparEstimator
> 
> > ```
> 
> Thanks for your continuous assistance, but this is still not working :(
> 
> 
> 
> ```ImportError: cannot import name 'DeeparEstimator' from 'gluonts.mx' (C:\Users\abcde\Anaconda3\envs\gluontsenv\lib\site-packages\gluonts\mx\__init__.py)```
> 
> 
> 
> 

Sorry, the A in DeepAR needs to be uppercase.

**Comment by RashidBakirov** [comment]
Thanks, both of the suggestions above worked and I have managed to run the example successfully.
